mne/viz/tests/test_evoked.py

```diff
@@ -323,6 +323,12 @@ def test_plot_compare_evokeds():
     figs = plot_compare_evokeds(evoked_dict, axes='topo', legend=True)
     for fig in figs:
         assert len(fig.axes[0].lines) == len(evoked_dict)
+    # test with (fake) CSD data
+    csd = _get_epochs(picks=np.arange(315, 320)).average()  # 5 EEG chs
+    for entry in csd.info['chs']:
+        entry['coil_type'] = 6
+        entry['unit'] = 117
```

Comment:
can you use more explicit FIFF constants here?

Reply:
> can you use more explicit FIFF constants here?

sure, will do.